samples/ResourceOwnerPasswordFlow/.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig is awesome:
```

Comment:
Hum, then I guess it's better to remove it from this PR for now.
We can always add `.editorconfig` everywhere in another PR if you think that's a nice addition :ghost: